drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -11,8 +11,33 @@ class HmisSchema::Project < Types::BaseObject
     description 'HUD Project'
     field :id, ID, null: false
     field :project_name, String, method: :ProjectName, null: false
-    field :project_type, Types::HmisSchema::ProjectType, method: :ProjectType, null: false
+    field :project_type, Types::HmisSchema::Enums::ProjectType, method: :ProjectType, null: true
     field :organization, Types::HmisSchema::Organization, null: false
+    field :operating_start_date, GraphQL::Types::ISO8601Date, null: false
+    field :operating_end_date, GraphQL::Types::ISO8601Date, null: true
+    field :description, String, null: true
+    field :contact_information, String, null: true
+    field :housing_type, Types::HmisSchema::Enums::HousingType, null: true
+    field :tracking_method, Types::HmisSchema::Enums::TrackingMethod, null: true
+    field :target_population, HmisSchema::Enums::TargetPopulation, null: true
+    field :HOPWAMedAssistedLivingFac, HmisSchema::Enums::HOPWAMedAssistedLivingFac, null: true
+    yes_no_missing_field :continuum_project
+    yes_no_missing_field :residential_affiliation
+    yes_no_missing_field :HMISParticipatingProject
+
+    # rubocop:disable Naming/MethodName
+    def HMISParticipatingProject
```

Comment:
do we like this?  Would it be better as `hmis_participating_project`?  This feels odd to me, but if we think it necessary, then that's fine.

Same goes for `field :HOPWAMedAssistedLivingFac` above.

Reply:
The method name needs to match the field name for ruby-graphql to know how to resolve that field. That is, when resolving the `HMISParticipatingProject` field, it would call a method called `HMISParticipatingProject` on the object type. It that isn't present, then it will call that method on the model, and so on down its chain.

I believe we can tell it to use an alternate method name, but I like that less than disabling rubocop here for the sake of the names matching the fields. I also don't know how providing an alternate method name would affect the chain of things that ruby-graphql will try to do to resolve the field in absence of a resolver method on the object type, so my preference is to let it do its default thing and make the names match.

Reply:
Also, FWIW these methods would pretty much only ever be called internally within this object type. So we're not exposing API that other things will rely on. If that was the case, then I agree that inconsistent method naming would not be good.